src/ui/public/notify/partials/toaster.html

```diff
@@ -48,6 +46,14 @@
             ng-class="'btn-' + notif.type"
             ng-click="notif.address()"
             >Fix it</button>
+          <button
+            type="button"
+            class="btn"
+            ng-repeat="action in notif.customActions"
+            ng-class="'btn-' + notif.type"
```

Comment:
I plan to have a follow-up PR that will allow custom control over the truncation length of the message. I can add a minor clean-up task there in that PR. 

It could also be a standalone PR though - no reason for it not to be